ui/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.test.js

```diff
@@ -125,4 +136,30 @@ describe('Confirm Page Container Content', () => {
     fireEvent.click(cancelButton);
     expect(props.onCancel).toHaveBeenCalledTimes(1);
   });
+
+  it('render contract address name from addressBook in title for contract', async () => {
+    props.hasSimulationError = false;
+    props.disabled = false;
+    props.toAddress = '0x06195827297c7A80a443b6894d3BDB8824b43896';
+    props.transactionType = TRANSACTION_TYPES.CONTRACT_INTERACTION;
+    const { queryByText } = renderWithProvider(
+      <ConfirmPageContainerContent {...props} />,
+      store,
+    );
+
+    expect(queryByText('Address Book Account 1')).toBeInTheDocument();
+  });
+
```

Comment:
Could we add a test for `TRANSACTION_TYPES.TRANSFER` or `TRANSACTION_TYPES.TRANSFER_FROM` with an unknown address?